service/ingressconfig/v2/version_bundle.go

```diff
@@ -0,0 +1,36 @@
+package v2
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundle() versionbundle.Bundle {
```

Comment:
This is like in node-operator. We do not really use it for release aggregation but we should have it for our internal business. The component and changelog information are not that useful but so be it for now.